homeassistant/components/telegram_bot/__init__.py

```diff
@@ -311,10 +311,11 @@ def initialize_bot(p_config):
     proxy_url = p_config.get(CONF_PROXY_URL)
     proxy_params = p_config.get(CONF_PROXY_PARAMS)
 
-    request = None
     if proxy_url is not None:
-        request = Request(proxy_url=proxy_url,
+        request = Request(con_pool_size=4, proxy_url=proxy_url,
                           urllib3_proxy_kwargs=proxy_params)
+    else:
+        request = Request(con_pool_size=4)
```

Comment:
Can you explain this change?

Reply:
To fix [warning from python-telegram-bot](https://github.com/python-telegram-bot/python-telegram-bot/blob/439790375ed8ed493c43e464aa8e2b60a77939db/telegram/ext/updater.py#L115)